### PR TITLE
Interface: Several changes to newcharacter window

### PR DESCRIPTION
**Pull request by Pupsi-Mupsi**
#### Summary

`SUMMARY: Interface "Several changes to newcharacter window"`

#### Purpose of change

This is just an idea for UI consistency as suggested in #12112.

"Changelog"

- Move the help text down and add sourcecode comment
- Expand and clarify help text
- Allow gender to be edited /changed via UP / DOWN and / or popup input 
- Rearrange editable values and fix broken line on the right
- Add space to "netPointCost" to match all other point-costs
- Change vehicle name from c_light_green to c_white
- Match punctuation where needed
- Astyle

<br />
Hope you like it.

---

#### Screenshot(s)

Before:

grafik

After:

grafik

### CODE REVIEW

**Comment by anothersimulacrum** [comment]
I like it.

**Comment by Pupsi-Mupsi** [comment]
Thanks!

**Comment by Leland** [comment]
Any chance you could list off the actual changes in the PR description? I see a few in the screenshots but feel like I'm playing spot the differences, hah.

**Comment by Pupsi-Mupsi** [comment]
Sure.

File: src/newcharacter.cpp [hunk 1]
```diff
@@ -2747,6 +2768,13 @@ tab_direction set_description( avatar &you, const bool allow_reroll,
                         you.blood_rh_factor = false;
                     }
                     break;
+                case char_creation::GENDER:
+                    if( you.male ) {
+                        you.male = false;
+                    } else {
+                        you.male = true;
+                    }
```
Comment (AMurkin) [inline src/newcharacter.cpp:2776]: Why not like this?
```suggestion
                    you.male = !you.male;
```

Comment (Pupsi-Mupsi) [reply src/newcharacter.cpp]: Thanks. Of course you're absolutely right.

**Pull request merged**